release-notes/System.ServiceModel.Duplex.md

```diff
@@ -8,8 +8,15 @@
 
 ## Version History
 
+### 4.0.1-rc2-24027
+
+* [NuGet Package](https://www.nuget.org/packages/System.ServiceModel.Duplex/4.0.1-rc2-24027)
+* [Release tag](https://github.com/dotnet/wcf/tree/v1.0.0-rc2)
```

Comment:
Just checking -- this rc2 release URL gives a 404 till you post the version currently in Draft mode.  As expected, correct?